controllers/metallb_controller.go

```diff
@@ -109,6 +110,8 @@ func (r *MetallbReconciler) syncMetalLBResources(config *metallbv1alpha1.Metallb
 		return err
 	}
 
+	override.ResourceOverrides(objs, config)
```

Comment:
Can you use the template engine instead of overriding after rendering?
Same thing we do with images.
Also, I am wondering if this will have any impacts on the RBACs